### PR TITLE
Fix Stream load double NumberTotalRows

### PR DESCRIPTION
**Pull request by kangkaisen**
For https://github.com/apache/incubator-doris/issues/1663

### CODE REVIEW

File: be/src/exec/broker_scan_node.cpp [hunk 1]
```diff
@@ -433,7 +433,6 @@ void BrokerScanNode::scanner_worker(int start_idx, int length) {
     }
 
     // Update stats
-    _runtime_state->update_num_rows_load_total(counter.num_rows_total);
```
Comment (imay) [inline be/src/exec/broker_scan_node.cpp]: If you remove this, rows filtered in this class will not be counted in total loaded rows.

Comment (kangkaisen) [reply be/src/exec/broker_scan_node.cpp]: Update

**Review by imay — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**